src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1413,6 +1478,8 @@ private void CleanUp()
 
             _detailModel.Dispose();
             _packageList.SelectionChanged -= PackageList_SelectionChanged;
+
+            _controlClosing?.Invoke(this, EventArgs.Empty);
```

Comment:
Not an expert, but I believe you need to unregister the event handler when the window is being disposed.

```suggestion
            _controlClosing?.Invoke(this, EventArgs.Empty);
            _controlClosing -= PackageManagerControl_ControlClosing;
```

... or you can send telemetry here and remove the event handler.

```suggestion
            TelemetryActivity.EmitTelemetryEvent(new PackageManagerCloseEvent(_sessionGuid, Model.IsSolution, _topPanel.Filter, _installedTabTelemetryData));
```